Ports/AvailablePorts.md

```diff
@@ -2,226 +2,227 @@
+| [`SDL2_ttf`](SDL2_ttf/)                   | SDL2\_ttf (TrueType Font add-on for SDL2)                       | 2.0.18                   | https://www.libsdl.org/projects/SDL_ttf/                                       |
+| [`sed`](sed/)                             | GNU sed                                                         | 4.8                      | https://www.gnu.org/software/sed/                                              |
+| [`sfinx`](sfinx/)                         | Sfinx                                                           | 1.1                      | https://www.scummvm.org/games/#games-sfinx                                     |
+| [`sl`](sl/)                               | Steam Locomotive (SL)                                           |                          | https://github.com/mtoyoda/sl                                                  |
+| [`soltys`](soltys/)                       | Soltys                                                          | 1.0                      | https://www.scummvm.org/games/#games-soltys                                    |
+| [`sqlite`](sqlite/)                       | SQLite                                                          | 3380200                  | https://www.sqlite.org/                                                        |
+| [`stpuzzles`](stpuzzles/)                 | Simon Tatham's Portable Puzzle Collection                       |                          | https://www.chiark.greenend.org.uk/~sgtatham/puzzles/                          |
+| [`stress-ng`](stress-ng/)                 | stress-ng                                                       | 0.13.12                  | https://github.com/ColinIanKing/stress-ng                                      |
+| [`Super-Mario`](Super-Mario/)             | Super-Mario Clone                                               |                          | https://github.com/Bennyhwanggggg/Super-Mario-Clone-Cpp                        |
+| [`tuxracer`](tuxracer/)                   | Tux Racer                                                       | 0.61                     | http://tuxracer.sourceforge.net/                                               |
+| [`tcl`](tcl/)                             | Tcl                                                             | 8.6.12                   | https://www.tcl-lang.org/                                                      |
+| [`termcap`](termcap/)                     | GNU termcap                                                     | 1.3.1                    | https://www.gnu.org/software/termutils/                                        |
+| [`thesilversearcher`](thesilversearcher/) | The Silver Searcher: A fast code-searching tool                 | 2.2.0                    | https://github.com/ggreer/the_silver_searcher                                  |
```

Comment:
Ah,, the big reformatting is unfortunate, but no way around it. Could you add a few more spaces (10?) here so the next port with one more character won't have to do the same?

Reply:
Yes, doing that now.